jbt-it/neomdb
# #306 events overview

How to test:
- go to Veranstaltungen and check if all are displayed properly
- check if you can create a new event
- check if the design is ok
